common/components/chrome/MainHeader.jsx

```diff
@@ -85,307 +52,129 @@ class MainHeader extends React.Component<{||}, State > {
-  }
-
-  _handleAlertClosing() {
-    let header = this.mainHeaderRef.current;
-    this._handleHeightChange(header.clientHeight - header.firstChild.clientHeight);
-  }
-
-  _renderLogInLink(): void {
-    return (
-      <SectionLink
-          activeSection={this.state.activeSection}
-          key="LogIn"
-          section={Section.LogIn}
-          title="Log In"
-        />
-    );
-  }
-
-  _renderAccountInfo(): void {
+  _renderNavBar() {
     return (
-      <div className="SubHeader-userImg-container">
-        <button
-          className="SubHeader-user-btn"
-          ref={node => { this.anchorEl = node }}
-          onClick={this._toggleDropdown}
-        >
-          {this._renderIcon()}
-        </button>
-        <Popper open={this.state.dropdown} anchorEl={this.anchorEl} transition>
-          {({ TransitionProps, placement }) => (
-            <Grow
-              {...TransitionProps}
-              id="menu-list-grow"
-              style={{ transformOrigin: 'top' }}
-            >
-              <Paper className="SubHeader-dropdown-menu">
-                <ClickAwayListener onClickAway={this._closeDropdown}>
-                  <MenuList>
-                    <p className="SubHeader-dropdown-name">{`${CurrentUser.firstName()} ${CurrentUser.lastName()}`}</p>
-                    <MenuItem onClick={(e) => this.navigateToSection(e, 'EditProfile')}>My Profile</MenuItem>
-                    {this.state.showMyProjects && <MenuItem onClick={(e) => this.navigateToSection(e, 'MyProjects')}>My Projects</MenuItem>}
-                  </MenuList>
-                  <Divider />
-                  <a href="/logout" onClick={this._closeDropdown}>
-                    <MenuItem>
-                      Sign Out
-                    </MenuItem>
-                  </a>
-                </ClickAwayListener>
-              </Paper>
-            </Grow>
-          )}
-        </Popper>
-      </div>
+      <Navbar collapseOnSelect expand="lg" bg="navlight" variant="light">
+        <Navbar.Brand><a href={url.section(Section.Home)}><img src={cdn.image("dl_logo.png")} alt="DemocracyLab" /></a></Navbar.Brand>
+        {CurrentUser.isLoggedIn() ? null : <Button className="MainHeader-showmobile MainHeader-login-button" variant="outline-primary" href={url.section(Section.LogIn, {prev: this._generatePrev()})}>Log In</Button>}
+        <Navbar.Toggle aria-controls="nav-pagenav-container" />
+        <Navbar.Collapse id="nav-pagenav-container" className="flex-column">
+          <Nav className="MainHeader-usernav ml-auto">
+            {CurrentUser.isLoggedIn() ? this._renderUserSection() : this._renderLogInSection()}
+          </Nav>
+          <Nav className="MainHeader-pagenav mr-auto">
+            <Nav.Link href={url.section(Section.Home)}>Home</Nav.Link>
+            <NavDropdown title="Projects" id="nav-projects">
+              <NavDropdown.Item href={url.section(Section.FindProjects)}>Find Projects</NavDropdown.Item>
```

Comment:
One thing we could do to more cleanly encapsulate conditional rendering for highlighting the current section is use a helper method for generating the Item.

renderNavDropdown(section, text) {
  // Return NavDropdown.Item that adds class with highlight if section == current section
}

Reply:
Added this in 8db706c though it could still use some further work.